src/js/components/help/__snapshots__/help.test.js.snap

```diff
@@ -1798,7 +1798,7 @@ exports[`Help Component static components renders Downloads correctly 1`] = `
                     class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiChip-deleteIcon MuiChip-deleteIconMedium MuiChip-deleteIconColorDefault MuiChip-deleteIconOutlinedColorDefault emotion-6"
                     data-testid="ArrowDropDownIcon"
                     focusable="false"
-                    value="https://downloads.mender.io/repos/debian/pool/main/m/mender/mender-client-dev_3.3.0-1%2Bubuntu%2Bfocal_amd64.deb"
+                    value="https://downloads.mender.io/repos/debian/pool/main/m/mender-client-dev/mender-client-dev_3.3.0-1%2Bubuntu%2Bfocal_amd64.deb"
                     viewBox="0 0 24 24"
```

Comment:
Could we have a similar test for `mender-configure-module` links? This can also be a tricky component to get the links right so it would be good to have a test for it.

Reply:
=> https://github.com/mendersoftware/gui/pull/3161/commits/9a2b27ef9ac5ab92b74f669e84190c1ee2ccf4c7#diff-8386ac26f9a9fce64463470bb86eac18f22b4c80f450109de912be9a7b5182ffR6-R9 👌